deps/pjsip/third_party/zsrtp/include/openssl_compat.h

```diff
@@ -0,0 +1,22 @@
+#ifndef _OPENSSL_COMPAT
```

Comment:
Any reason to keep compatibility with openssl-1.0?

Reply:
There are still distributions stuck on openssl-1.0 but personally, I need to keep 1.0 on my laptop for some Steam games while still being able to run Blink :)

Reply:
But you can have both 1.0 and 1.1 installed alongside each other and apps will use whatever they were compiled against since the libraries have different so names. What you cannot have installed simultaneously is the development versions of both 1.0 and 1.1, so you have to switch the development version based on what you compile.

Reply:
This is what I have installed right now on a debian system:

ii  libssl-dev:i386  1.1.1-2      i386         Secure Sockets Layer toolkit - development files
ii  libssl1.0.2:i386 1.0.2o-1     i386         Secure Sockets Layer toolkit - shared libraries
ii  libssl1.1:i386   1.1.1-2      i386         Secure Sockets Layer toolkit - shared libraries

Apps will use whichever version they were compiled against, but what I compile tries to build against 1.1

Reply:
True for most people, but not for my distro: exherbo. It's source-based and there hasn't been any work done (yet) on installing the two side-by-side. Most projects using openssl have maintained compatibility between the 2 versions, I'm sure 1.0 support will go away at some point but while it's still being actively maintained and there are people using it, I don't see why support for it should be dropped when it's possible to compile against either